package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@xt-pagesource/quality-control",
+  "version": "1.0.0",
+  "description": "XT Open Source Mono Repo",
+  "main": "index.js",
+  "author": "Sapient XT",
+  "license": "MIT",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pagesource/quality-control.git"
+  },
+  "publishConfig": {
+    "access": "restricted"
+  },
+  "scripts": {
+    "commit": "npx git-cz"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^7.0.0",
```

Comment:
We also need to think about creating packages for husky and commitlint.
That can be done on top of this later

Reply:
@ankurkakroo2  yes we can add for husky not sure about prettier really need separate package?

Reply:
Yes, we will need for husky, prettier and commitlint.

Reply:
@vinodloha  @ankurkakroo2  We can add more as much as packages to quality control but this comments should not be blocker for PR merge.